packages/components/src/components/carousel/carousel.jsx

```diff
@@ -73,37 +80,42 @@ const Carousel = ({
                     />
                 )}
                 <div className='dc-carousel__container'>
-                    {show_nav && nav_position === 'middle' && list.slice(computed_item_per_window - 1).length > 1 && (
-                        <span className='dc-carousel__icon' onClick={handlePrevClick}>
+                    {show_nav && nav_position === 'middle' && list_length > 1 && (
+                        <span
+                            className={classNames('dc-carousel__icon', { 'dc-carousel__icon--left': is_mt5 })}
+                            onClick={handlePrevClick}
+                        >
                             <Icon icon='IcChevronLeft' size='24' />
                         </span>
                     )}
 
                     <div
-                        className='dc-carousel__box'
+                        className={classNames('dc-carousel__box', {
+                            'dc-carousel__box--mt5': is_mt5 && list_length > 1,
+                        })}
                         style={{
                             width: `${computed_item_per_window * width}px`,
                         }}
                     >
                         <div
-                            className='dc-carousel__wrapper'
+                            className={classNames('dc-carousel__wrapper', { 'dc-carousel__wrapper--mt5': is_mt5 })}
                             style={{ transform: `translate3d(-${width * active_index}px, 0, 0)` }}
                         >
                             {list.map((type, idx) => (
-                                <Card key={idx} width={width}>
+                                <Card key={idx} width={width} is_mt5={is_mt5} list_length={list_length}>
```

Comment:
Is `Card` component accepting `is_mt5` as a prop? I don't see that added in this PR.

Reply:
Thank you. I haven't noticed it should be removed.